frontend/src/Style/Color.elm

```diff
@@ -0,0 +1,63 @@
+module Style.Color exposing (Color(..), background, border, text)
+
```

Comment:
It might make more sense to do the following:

* Move the text/backgorund/border functions to their respective modules. `Style.Background` doesn't exist, but as there are other CSS background properties, I could see that expanding in the future.
* Drop the custom type, use functions instead.